gdk-pixbuf/sys/src/lib.rs

```diff
@@ -720,6 +921,12 @@ extern "C" {
         error: *mut *mut glib::GError,
     ) -> gboolean;
 
+    //=========================================================================
+    // GdkPixbufNonAnim
```

Comment:
I guess this can simply be added to `gdk-pixbuf/Gir.toml` into the `generate` section :)

Reply:
Which version was that added though?

Reply:
No sure exactly, but I seem them in a revision from 2001, so I assume they have been there long enough :laughing: 

I will add to `generate`

Reply:
Curious that it wasn't part of the gir before

Reply:
Please also update the `gir-files` submodule to the merged version, and then it should all be good to be merged

Reply:
Update with the submodule version, removing WIP.

With regard to `PixbufNonAnim` I added the generate for `new`, but that did not seem to do much... it only results in these two lines:

```
mod pixbuf_non_anim;
pub use self::pixbuf_non_anim::PixbufNonAnim;
```

Is that ok? Also, I did not at `get_type` since I assumed that is kind of special and I did not see in the `generate` section of other objects...

Unrelated to this PR, when running the generator, there seems to be a spurious change in `bindings.rs`, I removed that manually, but it sounds like something is out of sync...

Reply:
I dropped the `generate` stuff... it fails the build and clearly I do not know what I am doing and trying blindly. I guess it can be fixed it its own patch

Reply:
OK, no problem :)